lightning/src/ln/channel.rs

```diff
@@ -5641,6 +5696,9 @@ impl<SP: Deref> OutboundV1Channel<SP> where SP::Target: SignerProvider {
 
 		let temporary_channel_id = ChannelId::temporary_from_entropy_source(entropy_source);
 
+		let next_per_commitment_point = holder_signer.get_per_commitment_point(INITIAL_COMMITMENT_NUMBER, &secp_ctx)
+			.map_err(|_| APIError::ChannelUnavailable { err: "Unable to generate initial commitment point".to_owned()})?;
```

Comment:
here and elsewhere would be good if either the specific signer error was included, or this said something about the signer not being available - otherwise it might be puzzling for the dev.

Reply:
Yep... as I mentioned in the Discord chat I haven't gotten to several of these. Right now, they're basically just modified to handle the changes to the signer's signature.